submissions/Shrai-dev/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,344 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movie Seat Booking</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans&family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <main class="main">
+      <section class="movie__info">
```

Comment:
If you use BEM methodology, in this line you have BEM element `.movie_info` outside BEM block `.movie`.
You can't use BEM element outside BEM block. [Read more.](https://ru.bem.info/methodology/quick-start/#%D1%8D%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82)